admin/acceptance.py

```diff
@@ -220,38 +227,47 @@ def stop_nodes(self):
                 print "Failed to destroy %s: %s" % (node.name, e)
 
 
-def rackspace_runner(config, **kwargs):
+def runner_for_cloudprovisioner(provisioner_name):
     """
-    Run the tests against rackspace nodes.
+    Dynamically generate and return a factory function which will be used to
```

Comment:
This functions feels like a runner factory factory. I think one of the layers should be removed.